chore: update main upgrade doc to include cosmovisor (#270)
axelarnetwork/axelar-docs: pages/resources/mainnet/upgrades/v28.md

Upgrade height `4984950` [countdown](https://www.mintscan.io/axelar/blocks/4984950)

> If you're using cosmovisor to manage upgrades, please follow this guide [here](https://polkachu.com/chain_upgrades/121) instead

Reviewer: ```suggestion
If you're using `cosmovisor` to manage upgrades, you can follow the guide [here](https://polkachu.com/chain_upgrades/121) instead.
```
